.github/workflows/breaking-change-review.yml

```
@@ -0,0 +1,145 @@
+name: PR Review Workflow for Breaking Changes
```

Shorter name would be better I think
```suggestion
name: PR Review Workflow
```